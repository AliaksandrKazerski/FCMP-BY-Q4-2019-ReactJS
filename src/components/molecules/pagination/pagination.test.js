import React from 'react';
import { shallow } from 'enzyme';

import Pagination from './index';

describe('Pagination', () => {
  it("Pagination should render correctly", () => {
  const myComponent = shallow(
    <Pagination/>
  );
  
  expect(myComponent).toMatchSnapshot()
  });
});