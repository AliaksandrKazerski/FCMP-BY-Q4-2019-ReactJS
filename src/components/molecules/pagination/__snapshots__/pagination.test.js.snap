// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination Pagination should render correctly 1`] = `ShallowWrapper {}`;
